src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -44,14 +44,15 @@ function ProductDetails({ product: staleProduct }: Props) {
       isVariantOf: { name, productGroupID: productId },
       image: productImages,
       offers: {
-        offers: [{ price, listPrice, seller }],
+        offers: [{ availability, price, listPrice, seller }],
         lowPrice: spotPrice,
       },
       breadcrumbList,
     },
   } = data
 
   const { currency } = useSession()
+  const buyDisabled: boolean = availability !== 'https://schema.org/InStock'
```

Comment:
Interesting!
https://github.com/vtex/faststore/blob/653c9d8e54558c19b9ee7aa8c3387d56ed2d16bb/packages/api/src/platforms/vtex/resolvers/offer.ts#L14-L17

Reply:
I think we don't need the `boolean` type here due to the implicit type.